libeventheader-decode-cpp/src/EventFormatter.cpp

```diff
@@ -297,7 +297,7 @@ class StringBuilder
     // Requires: valSize != 0.
     // Writes: valSize * 3 - 1, e.g. [00 11 22].
     void
-    WriteHexBinary(void const* val, size_t valSize) noexcept
+    WriteHexBytes(void const* val, size_t valSize) noexcept
```

Comment:
Note: The EventHeader "hex_binary" type was renamed to "hex_bytes" a long time ago, but these private function names kept the old naming. Fixed.